views/template.html

```diff
@@ -70,7 +70,7 @@
       * { padding: 0; margin: 0; box-sizing: border-box; }
       html, body, pre {
         width: 100%;
-        height: 100%;
+        min-height: 100%;
```

Comment:
ok so I'm kind of worried that this would still breaks? but let's try this because I also don't know